docs/customization/customizing-the-backend.mdx

```diff
@@ -336,6 +336,82 @@ found in the locale file.
 * `partial:` can be used in complex scenarios when you want a partial to be rendered as content
 within your menu item. For instance, `partial: 'spree/admin/orders/rejected_orders'`.
 
+## Customizing assets
+
+Solidus leverages the [Rails asset
+pipeline](https://guides.rubyonrails.org/asset_pipeline.html) to allow for
+customization and overriding of your backend assets. We recommend that you
+familiarize yourself with the Rails asset pipeline before going any further.
+
+Some aspects of the style in the backend have been extracted into SCSS
+variables. You can look at them in the [`spree/backend/globals/_variables.scss`
+file](https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/globals/_variables.scss).
+If you want to redefine some of them, you can create a
+`spree/backend/globals/_variables_override.scss` file under your application
+styles. For instance, if you wanted to make the header black:
```

Comment:
I think `styles` is not the right word here, or am I missing the sense of the sentence?

Reply:
Oh, thanks for spotting it. Fixed! (`styles` -> `stylesheets directory`).